service/src/test/java/org/kiwiproject/champagne/resource/filter/DeployableSystemResponseFilterTest.java

```diff
@@ -1,21 +1,21 @@
 package org.kiwiproject.champagne.resource.filter;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.mock;
-
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.kiwiproject.champagne.model.DeployableSystemThreadLocal;
 
 import javax.ws.rs.container.ContainerRequestContext;
 import javax.ws.rs.container.ContainerResponseContext;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
 @DisplayName("DeployableSystemResponseFilter")
 class DeployableSystemResponseFilterTest {
 
     @Test
     void shouldClearDeployableSystem() {
-        DeployableSystemThreadLocal.setCurrentDeployableSystem(1L);
+        DeployableSystemThreadLocal.setCurrentDeployableSystem(1L, false);
```

Comment:
After seeing all the `boolean` arguments, would it be better to use an `enum` for the user type? e.g. something like:

```java
DeployableSystemThreadLocal.setCurrentDeployableSystem(1L, UserType.ADMIN);
```

Reply:
I know you hate boolean parameters. I'll create an issue to relook at this construct.